cpp/ql/test/library-tests/ir/ir/ir.cpp

```diff
@@ -1055,28 +1055,75 @@ void Lambda(int x, const String& s) {
   lambda_inits(6);
 }
 
-template<typename T>
-struct vector {
-    struct iterator {
-        T* p;
-        iterator& operator++();
-        T& operator*() const;
+namespace std {
```

Comment:
Can we indent the following more sensibly? It's a bit of a mess at the moment (probably because spaces and tabs are mixed?).

Reply:
Converted to spaces (the one true indentation!) in 51db2b0bc4a6f6fbe25cc2f75a1a529a389462ac